plugin/pkg/scheduler/scheduler.go

```diff
@@ -50,12 +52,14 @@ type Scheduler struct {
 	config *Config
 }
 
+// StopEverything when called, closes the StopEverything channel to shut down
```

Comment:
StopEverything closes the scheduler config's StopEverything channel, to shut down the Scheduler.